x/upgrade/module.go

```diff
@@ -223,6 +226,10 @@ func ProvideModule(in ModuleInputs) ModuleOutputs {
 	m := NewAppModule(k, in.AddressCodec)
 	gh := govv1beta1.HandlerRoute{RouteKey: types.RouterKey, Handler: NewSoftwareUpgradeProposalHandler(k)}
 
+	if upgradePlan, err := k.ReadUpgradeInfoFromDisk(); err == nil {
+		telemetry.SetGaugeWithLabels([]string{"server", "info"}, 1, []metrics.Label{telemetry.NewLabel("upgrade_height", strconv.FormatInt(upgradePlan.Height, 10))})
```

Comment:
```suggestion
	if upgradePlan.Height > 0 {
		telemetry.SetGaugeWithLabels([]string{"server", "info"}, 1, []metrics.Label{telemetry.NewLabel("upgrade_height", strconv.FormatInt(upgradePlan.Height, 10))})
	}
```

We need this because the app is instantiated the first time with a temporary directory, for getting the application encoding (https://github.com/cosmos/cosmos-sdk/blob/c1ea84d583499abccdc068c7b0e69567ad899107/simapp/simd/cmd/root.go#L47)